modules/couchbase/src/main/java/org/testcontainers/couchbase/CouchbaseContainer.java

```diff
@@ -150,18 +150,28 @@ private void startProxy(String networkAlias) {
                 proxy.addExposedPort(port.getOriginalPort());
             }
         }
+
         proxy.setWaitStrategy(null);
         proxy.start();
-    }
 
-    private void exposePortThroughProxy(String networkAlias, int originalPort, int mappedPort) {
         ExecCreateCmdResponse createCmdResponse = dockerClient
             .execCreateCmd(proxy.getContainerId())
-            .withCmd("/usr/bin/socat", "TCP-LISTEN:" + originalPort + ",fork,reuseaddr", "TCP:" + networkAlias + ":" + mappedPort)
+            .withCmd(
+                "sh",
+                "-c",
+                Stream.of(CouchbasePort.values())
+                    .map(port -> {
+                        return "/usr/bin/socat " +
+                            "TCP-LISTEN:" + port.getOriginalPort() + ",fork,reuseaddr " +
+                            "TCP:" + networkAlias + ":" + getMappedPort(port);
+                    })
+                    .collect(Collectors.joining(" & ", "true", ""))
+            )
             .exec();
 
         dockerClient.execStartCmd(createCmdResponse.getId())
-            .exec(new ExecStartResultCallback());
+            .exec(new ExecStartResultCallback())
+            .awaitCompletion(10, TimeUnit.SECONDS);
```

Comment:
FYI we were missing `awaitStarted` here, and I guess it caused some races

Reply:
Good catch!